config/config-example.js

```diff
@@ -413,18 +413,24 @@ exports.disablehotpatchall = false;
  *     - hotpatch: /hotpatch, /crashfixed and /savelearnsets commands.
```

Comment:
So where it currently says:

```
 *   All the possible permissions are as follows:
```

```
 *
 * Permissions are a string key such as `mute` followed by a boolean. `true` means group
 * members have that permission, `false` means they don't, and `undefined` means permission
 * is inherited from the `inherit` group.
 * 
 * Some permissions, such as mute/ban, have a target (user you're doing the action to). These,
 * when set to `true`, will use the jurisdiction list in the `jurisdiction` key to determine whether
 * or not you have permission to do it to that user. You can also set them to a string to use that
 * string as a custom jurisdiction list, replacing the one in the `jurisdiction` key.
```

At which point, on the individual permissions, you can just say "has a target".